lms/djangoapps/bulk_email/models.py

```diff
@@ -91,7 +92,15 @@ def create(cls, course_id, sender, to_option, subject, html_message, text_messag
             text_message = html_to_text(html_message)
 
         # perform some validation here:
-        if to_option not in TO_OPTIONS:
+        if to_option.isdigit():
+            if not GroupedQuery.objects.filter(id=int(to_option)).exists():
+                message = "Course email for \"{course}\" being sent to query id that does not exist: {query_id}, subject \"{subject}\"".format(
```

Comment:
Nit: I'll often use single quotes if I need to insert quotes w/in a double-quoted string.

```python
message = "Course email for '{course}' ..."
```